lib/ABI/FunctionType.cpp

```diff
@@ -130,6 +154,8 @@ class ConversionHelper {
   using DistributedArguments = llvm::SmallVector<DistributedArgument, 4>;
 
   using ArgumentContainer = SortedVector<model::Argument>;
+  using NewTypeContainer = llvm::SmallVector<UpcastablePointer<model::Type>,
+                                             16>;
```

Comment:
Can we call this `TypeVector`? I'm not sure what `New` means here.

Reply:
New as in `opposite-of-old`, the replacement.

Either way, `TypeVector` works.

Reply:
Renamed it into `TypeVector` (and also `ArgumentContainer` next to it into `ArgumentVector` for consistency).